src/controllers/request-controller.ts

```diff
@@ -13,10 +13,8 @@ import {
   RequestAnchorParams,
 } from '../ancillary/anchor-request-params-parser.js'
 import bodyParser from 'body-parser'
-import { isLeft } from 'fp-ts/lib/Either.js'
-import { makeErrorMessage } from '../ancillary/throw-decoder.js'
-import * as t from 'io-ts'
-import * as te from '../ancillary/io-ts-extra.js'
+import * as t from 'codeco'
```

Comment:
Similar to previous comment, can we get rid of `*` import or name something else?